src/main/java/xRoss.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+
+public class xRoss {
+
+    // prints a divider line and new line to command line output
+    public static void printDividerLine() {
+        System.out.println("..........................................................................................");
+    }
+
+    // prints welcome message
+    public static void printWelcomeMessage() {
+        String logo = "        _____\n"
+                + "       |  __ \\ _  __  __\n"
+                + " _  _  |    _// \\|  \\|  \\\n"
+                + "\\ \\/ / | |\\ \\| | |\\ \\\\ \\\n"
+                + "/_/\\_\\ |_| \\_\\\\_/\\__|\\__|\n";
+        System.out.println("Hello from\n" + logo);
+
+        printDividerLine();
+        System.out.println("\tHello! I'm xRoss, your personal chat bot assistant!\n\tWhat can I do for you today?\n");
+        printDividerLine();
+    }
+
+    // prints exit message
+    public static void printExitMessage() {
+        printDividerLine();
+        System.out.println("\tBye!\n\tHave a nice day and I hope to see you again soon!\n");
+        printDividerLine();
+    }
+
+    public static void printEcho(String inputLine) {
+        printDividerLine();
+        System.out.println("\t" + inputLine);
+        System.out.println("\tI did not quite understand what you meant there, so I'll just echo your input...\n");
+        printDividerLine();
+    }
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+
+        // TaskManager instance to keep track of all tasks
+        TaskManager taskManager = new TaskManager();
+
+        // setting up variable and scanner for user input
+        String inputLine;
+        Scanner in = new Scanner(System.in);
+
+        // boolean value on whether
+        boolean continueLoop = true;
```

Comment:
maybe you could consider a more boolean sounding name here?

Reply:
I would like to suggest some more boolean sounding names such as canLoop, isContinuing etc.